server/src/main/java/org/elasticsearch/node/Node.java

```diff
@@ -148,6 +152,12 @@ public class Node implements Closeable {
         Property.NodeScope
     );
 
+    public static final Setting<TimeValue> SHUTDOWN_SEARCH_TIMEOUT_SETTING = Setting.positiveTimeSetting(
+        "node.async_search_grace_period",
+        TimeValue.timeValueMinutes(1),
```

Comment:
This seems like an aggressive default. For http we defaulted to infinite. Could we use that instead, letting operators decide the appropriate timeout if they want to timeout?

Reply:
I thought I recalled a conversation where we wanted an agressive timeout here, but upon reflection that might have been specific to our environment. I'll make this match the HTTP timeout default.